### ISSUE TITLE
Removing one item does not apply on the server

### ISSUE THREAD

**Issue by Yyuzu** [GroundApps/ShoppingList#58]
Hi,

Whenever I click on an item, it gets crossed in the app, but no query is sent to the server. When I close the app and come back nothing gets crossed / deleted. Is it normal ?

I'm using the app from F-Droid v1.3 (4) with the backend on my server (master branch).

Thank you for the app :-)
Regards

**Comment by ghost** [comment]
Cloud you please post your log from server to see the request

**Comment by J-8** [comment]
Hey, 
thanks for taking the time to comment! 

Currently this behavior is not supported, simply because there has not been time to implement it. But it is something that we hope to add with the next update!

**Comment by Yyuzu** [comment]
Hey ;-)

The line gets crossed when I click on it in the app, but there is no request towards the server (thus I cannot copy the access.log line ^^).

I'm a bit confused, if it is not yet supported why does it gets crossed in the app ?

**Comment by J-8** [comment]
Hey!

The crossing on the list was not meant to by synchronized. I never thought of it until some people commented on it. You are not the first one;)
So right now when you cross an item, you can delete it from the list with the top right button (the two checkmarks). All crossed items get deleted.

The problem is that when first writing the app, I had a very limited view on what things would be useful.

**Comment by ghost** [comment]
@Kyoshiro-san 
We would write the whole app new.
So the next version will worked better

**Comment by Yyuzu** [comment]
Oh I see ! I didn't understand what the checkmarks button was for, now it's clear ^^

Well, for a personal usage it's ok but I'd like to use it with my partner, and it could be useful to know if one of us has already bought something. If the "mark done" is synced it'd achieve that, while we might forget something even was on the list if it gets removed. [I don't know if I'm being clear right now ^^']

Anyways, that's a nice app and idea, I understand it's in its early stages, it's good to know it's under heavy thinking and I wish the best for you and the project :-)

I might post a few suggestions if you don't mind !

**Comment by J-8** [comment]
You are absolutely right synchronization for a use with someone else does make more sense.

Thanks for you kind words and keep suggestions coming!

**Comment by Yyuzu** [comment]
Will do !
I might be able to help, if you need some ^^

**Comment by J-8** [comment]
Sure! Check back next week when we have updated the road map.

**Comment by Yyuzu** [comment]
Ok ^^
